ReText/__main__.py

```diff
@@ -100,6 +104,20 @@ def main():
 		app.setStyleSheet(QTextStream(sheetfile).readAll())
 		sheetfile.close()
 	window = ReTextWindow()
+
+	connection = QDBusConnection.sessionBus()
+	if connection.isConnected() and not parser.isSet(newWindowOption):
+		connection.registerObject('/', window, QDBusConnection.ExportAllSlots)
```

Comment:
so you are exposing all of them

Reply:
Yes, I am still exposing all of them, and taking an opportunity to use this, by calling the `raise()` method.